### PR TITLE
fixing path for dist

### PR DESCRIPTION
**Pull request by daudich**
The new `demo.html` is being created in `/example` but the `/dist` file is one directory up.

### CODE REVIEW

File: example/template/demo.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <meta charset="utf-8">
     <title>Conversation Demo</title>
     <script src="http://code.jquery.com/jquery-1.11.3.min.js"></script>
-    <script src="/dist/WIDGET_CODE"></script>
```
Comment (juliangarritano) [inline example/template/demo.html]: This works fine with http://localhost:8282/example/demo, but your fix will work with the file:// protocol as well.

**Comment by juliangarritano** [comment]
:+1:

**Comment by lemieux** [comment]
How did we never catch that?

Marc-Antoine Lemieux

2015-07-10 16:10 GMT-04:00 juliangarritano <notifications@github.com>:

**Comment by daudich** [comment]
So, should I merge it then?

**Comment by lemieux** [comment]
Yep go for it

**Comment by juliangarritano** [comment]
We actually did catch that, I just started using localhost and forgot about it 😜

**Pull request merged**